### ISSUE TITLE
CR Request for <WAI-Adapt: Content Module>

### ISSUE THREAD

**Issue by ruoxiran** [w3c/transitions#443]
# Document title, URLs, estimated publication date
[
Document title: WAI-Adapt: Content Module (We changed the document name once before from Personalization Content Module 1.0 to WAI-Adapt: Content Module, and the name of Task Force was also changed from Personalization to WAI-Adapt)
URLs: [https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/snapshot/CRD-adapt-content-20220726.html ](https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/snapshot/CRD-adapt-content-20220726.html )
estimated publication date: 2022-07-26 （This is the ADA anniversary date and APA hopes to publish the document on this date, but if the process is tight and we can't, we will postpone the publication.）
]

# Abstract
[
This specification provides web content authors a standard approach to support web users who are persons with various cognitive and learning disabilities, including users who:
Customarily communicate using symbolic languages generally known as Augmentative and Alternative Communications (AAC);
Need more familiar icons (and other graphical symbols) in order to comprehend page content;
Benefit from additional help resources;
Require a simplified page in order to interact successfully.
The technology described in this specification is intended to be used to programmatically transform the appearance of typical web content including form controls, icons, and other user interface elements into a rendering more familiar and comprehensible to an individual user. Providing additional help is supported, as is automated page simplification. Use cases and relevant vocabulary terms are defined which allow user agents to augment or adapt content to better fit a particular user's particular needs. This helps users with varying needs to understand web content more readily, by simplifying the web interaction, and by avoiding forcing the user to grapple with unfamiliar representations they are ill equipped to comprehend.

This WAI-Adapt: Content Module is a component of the WAI-Adapt series introduced in the WAI-Adapt Explainer.
]

# Status

Please note that we have added a paragraph about feedback in status, please help to review.
[[https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/snapshot/CRD-adapt-content-20220726.html#sotd](https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/snapshot/CRD-adapt-content-20220726.html#sotd)]

# Link to group's decision to request transition
[[https://lists.w3.org/Archives/Public/public-apa-admin/2022Jul/0022.html](https://lists.w3.org/Archives/Public/public-apa-admin/2022Jul/0022.html)]

# Changes
[Please find changes in adapt github commits: https://github.com/w3c/adapt/commits/main/content]

# Requirements satisfied
[we have published a draft note for requirements of the document, sections 2.1 to 2.4 related to this publication. [https://www.w3.org/TR/adapt-requirements/](https://www.w3.org/TR/adapt-requirements/)]

# Dependencies met (or not)
[
[https://www.w3.org/2021/08/apa-wg-charter#coordination](https://www.w3.org/2021/08/apa-wg-charter#coordination)
this is the dependencies met for the whole APA WG, with regard to WAI-Adapt TF, we had liaisons with i18n and Epub, and we still want to talk with WICG to seek a prefix of vocabulary after CR.
]

# Wide Review
[
TAG: closed, https://github.com/w3ctag/design-reviews/issues/476
i18n: closed, https://github.com/w3c/i18n-activity/issues/1545
a11y: closed, https://github.com/w3c/a11y-request/issues/15
Privacy and security: closed the wide review issue at: https://github.com/w3c/adapt/issues/131. But the relevant one at PING is still open, https://github.com/w3cping/tracking-issues/issues/245, I have confirmed with PLH during a meeting, and he mentioned we can continue the CR process. The document had a Privacy and Security Consideration section at [https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/snapshot/CRD-adapt-content-20220726.html#privacy-and-security-considerations](https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/snapshot/CRD-adapt-content-20220726.html#privacy-and-security-considerations)
]

# Issues addressed
[
[https://github.com/w3c/adapt/issues?q=is%3Aissue+is%3Aopen+label%3A%221%29+content+module%22](https://github.com/w3c/adapt/issues?q=is%3Aissue+is%3Aopen+label%3A%221%29+content+module%22)
Issues related to this publication have been closed, and two remaining issues are related to future implementation plans. 
]

# Formal Objections
[no objection]

# Implementation
[
WAI-Adapt is using "data-*" approach at the moment to gain implementation experience and demonstrate the usefulness of WAI-Adapt in practice. We hope to use this CR document publication to seek a new prefix to replace data- from W3C and WHATWG, then we will carry out the second CRD publication. See preliminary [implementation information in Explainer](https://www.w3.org/TR/adapt/#vocabulary-implementations).

There are some early [implementations of content](https://github.com/w3c/adapt/wiki/Implementations-of-Semantics) that can maximize the benefit for users, we have a plan to draft exit criteria.

minimum CR duration: 6 months
]

# Patent disclosures
[[https://www.w3.org/groups/wg/apa/ipr#disclosures](https://www.w3.org/groups/wg/apa/ipr#disclosures)]

**Comment by swickr** [comment]
We need CR exit criteria before we will approve the transition.

The [TAG review](https://github.com/w3ctag/design-reviews/issues/476) cited does not appear to have resolved the questions raised by the TAG.

Per 3.6.1 the BCI index is normatively referenced.  That should appear in References.

**Comment by ruoxiran** [comment]
reopen [issue #170](https://github.com/w3c/adapt/issues/170) to resolve the TAG review.

**Comment by plehegar** [comment]
On APA agenda for September 13

**Comment by plehegar** [comment]
At this time, should this transition be withdrawn?

**Comment by plehegar** [comment]
See also APA/TAG during TPAC2022: [Unblocking the WAI-Adapt Content Module 1.0 CR](https://www.w3.org/2022/09/13-apa-minutes.html#t01).

**Comment by ruoxiran** [comment]
> At this time, should this transition be withdrawn?

yes, I would like to close this transition. If there is any update I will recreate a new transition request.

**Comment by ruoxiran** [comment]
Follow up after TPAC 2022 at: https://lists.w3.org/Archives/Group/group-apa-chairs/2022Nov/0066.html
